airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/ticket_metrics.json

```diff
@@ -159,10 +159,14 @@
       "format": "date-time"
     },
     "updated_at": {
-      "description": "Timestamp when the ticket was last updated",
+      "description": "Datetime when the ticket last created a ticket event",
       "type": ["null", "string"],
       "format": "date-time"
     },
+    "generated_timestamp": {
```

Comment:
This is interesting, were we originally fetching this field for `ticket_metrics`, but just didn't have it listed in the schema?

I didn't see any changes to the stream that add this field so that's why I assumed it was already being sent even if it wasn't in the schema

Reply:
It's fetched for the `Tickets`/`TicketSubstream` streams but it's not included in the `TicketMetrics` endpoint at all, so I pass it down and add it in via `TicketMetrics.parse_response`.